app/templates/components/ui-table/cell/cell-tickets.hbs

```diff
@@ -1,9 +1,13 @@
-{{#if record.tickets}}
-  <div class="ui list">
-    {{#each record.tickets as |ticket|}}
-      <div class="item">{{ticket.name}} ({{ticket.orderStatistics.tickets.completed}}/{{ticket.quantity}}) <span class="item muted text">[{{ticket.type}}]</span></div>
-    {{/each}}
-  </div>
+{{#if record.ticketUrl}}
+  <div>{{t 'You have to go to '}}<a href="{{record.ticketUrl}}">{{record.ticketUrl}}</a>{{t ' to check ticket info'}}</div>
```

Comment:
Change the text to `Vist \<link\> to book tickets.`

Reply:
Will make certain changes.